newHometask/PersonRunner.java

```diff
@@ -0,0 +1,53 @@
+package newHometask;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class PersonRunner {
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.print("Enter the file path: ");
+        String filePath = reader.readLine();
+        reader.close();
+        List<Person> personList = new ArrayList<>();
+        BufferedReader fileReader = new BufferedReader(new FileReader(filePath));
```

Comment:
Where is the correct handling of exceptions?)
Eg. when the user entered incorrect file path